scripts/issue/247/sweep_strat_tokens.py

```diff
@@ -0,0 +1,148 @@
+TECH_OPS = registry.eth.badger_wallets.techops_multisig
+
+HARVEST_FORWARDER = registry.eth.harvest_forwarder
+
+PBTC_STRATEGY_DEP = ADDRESSES_ETH["strategies"]["_deprecated"]["native.pbtcCrv"]["v1"]
+PBTC_STRATEGY = ADDRESSES_ETH["strategies"]["native.pbtcCrv"]
+
+OBTC_STRATEGY_DEP = ADDRESSES_ETH["strategies"]["_deprecated"]["native.obtcCrv"]["v1"]
+OBTC_STRATEGY = ADDRESSES_ETH["strategies"]["native.obtcCrv"]
+
+OBTC_VAULT = ADDRESSES_ETH["sett_vaults"]["bcrvOBTC"]
+PBTC_VAULT = ADDRESSES_ETH["sett_vaults"]["bcrvPBTC"]
+
+PNT = registry.eth.treasury_tokens.PNT
+BOR = registry.eth.treasury_tokens.BOR
+BORING = registry.eth.treasury_tokens.BORING
+BADGER = registry.eth.treasury_tokens.BADGER
+
+STRATEGIES = [PBTC_STRATEGY_DEP, PBTC_STRATEGY, OBTC_STRATEGY_DEP, OBTC_STRATEGY]
+STRATEGY_MAPPING = {
+    PBTC_STRATEGY_DEP: "pbtcCrv_deprecated",
+    PBTC_STRATEGY: "pbtcCrv",
+    OBTC_STRATEGY_DEP: "obtcCrv_deprecated",
+    OBTC_STRATEGY: "obtcCrv",
+}
+
+TOKENS = [PNT, BOR, BORING]
+TOKEN_MAPPING = {
+    PNT: "PNT",
+    BOR: "BOR",
+    BORING: "BORING",
+}
+
+FEE = 0.2  # 20% to DAO
+
+
+def sweep_to_techops():
+    safe = GreatApeSafe(DEV_MULTI)
+    safe.take_snapshot(TOKENS)
+
+    for strategy in STRATEGIES:
+        console.print(f"[green]Sweeping {STRATEGY_MAPPING[strategy]}[/green]")
+        sweep_strategy(strategy, safe)
+
+    bor_contract = interface.ERC20(BOR, owner=safe.account)
+    boring_contract = interface.ERC20(BORING, owner=safe.account)
+    pnt_contract = interface.ERC20(PNT, owner=safe.account)
+
+    final_bor_bal = bor_contract.balanceOf(safe.address)
+    final_boring_bal = boring_contract.balanceOf(safe.address)
+    final_pnt_bal = pnt_contract.balanceOf(safe.address)
+
+    bor_contract.transfer(TECH_OPS, final_bor_bal)
+    boring_contract.transfer(TECH_OPS, final_boring_bal)
+    pnt_contract.transfer(TECH_OPS, final_pnt_bal)
+
+    assert 0 == bor_contract.balanceOf(DEV_MULTI)
+    assert 0 == boring_contract.balanceOf(DEV_MULTI)
+    assert 0 == pnt_contract.balanceOf(DEV_MULTI)
+
+    assert final_bor_bal == bor_contract.balanceOf(TECH_OPS)
+    assert final_boring_bal == boring_contract.balanceOf(TECH_OPS)
+    assert final_pnt_bal == pnt_contract.balanceOf(TECH_OPS)
+
+    console.print(f"BOR swept: {final_bor_bal / 10 ** 18}")
+    console.print(f"BORING swept: {final_boring_bal / 10 ** 18}")
+    console.print(f"PNT swept: {final_pnt_bal / 10 ** 18}")
+
+    safe.post_safe_tx(call_trace=True)
+
+
+def sell_for_badger():
+    safe = GreatApeSafe(TECH_OPS)
+    safe.init_cow()
+
+    sell_all_of_token(BOR, safe)
+    sell_all_of_token(BORING, safe)
+    sell_all_of_token(PNT, safe)
+
+    safe.post_safe_tx(call_trace=True)
+
+
+def distribute_to_users():
+    safe = GreatApeSafe(TECH_OPS)
+    forwarder = interface.IHarvestForwarder(HARVEST_FORWARDER, owner=safe.account)
+
+    obtc_amount = int(0 * (1 - FEE))  # replace 0 with amount after cowswaps
+    forwarder.distribute(BADGER, obtc_amount, OBTC_VAULT)
```

Comment:
as discussed in private i think we require prior to the `distribute` the approvals and also will be great to have the amounts coming from cowswap to be allowed rather thru args in this method to just introduce thru the cli to exec.so later the only element to sign-off are the figures coming from the swap itself